documentation/istio_keycloak_integration.asciidoc

```diff
@@ -0,0 +1,213 @@
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+== Authentication with Keycloak and Istio
+
+=== Introducing _https://www.keycloak.org/[Keycloak]_
+Keycloak is an opensource IAM (Identity and Access Management) solution with a broad set of features like SSO, authentication and authorization, social login, multifactor authentication etc.
+
+=== Introducing _https://istio.io/latest/docs/concepts/[Istio]_
```

Comment:
Same here.
IMHO this structure will not work out on the long run.
We rather need documentation and patterns for Keycloak and Istio separate from each other.